pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,127 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+
+def spectral_factor_pelland(airmass_absolute, clearsky_index,
```

Comment:
I think we should consider calling this function `spectral_factor_pvspec`, as "PVSPEC" is the name given by the model's authors.